Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8-jdk-alpine AS builder
+WORKDIR target/dependency
+ARG APPJAR=target/*.jar
+COPY ${APPJAR} app.jar
+RUN jar -xf ./app.jar
+
+FROM oracle/graalvm-ce:19.2.0.1 AS native
+RUN gu install native-image
+ARG DEPENDENCY=target/dependency
```

Comment:
The `.github/workflows/deploy-native-image.sh` script only does `mvn compile` (not eg `package`). Is that expected?

Reply:
changed it to `package`